docs/usage.rst

```diff
@@ -18,12 +18,14 @@ Current usage of ``pydocstringformatter``:
                                 [--capitalize-first-letter  --no-capitalize-first-letter]
                                 [--final-period  --no-final-period]
                                 [--quotes-type  --no-quotes-type]
+                                [--numpydoc-section-order  --no-numpydoc-section-order]
+                                [--numpydoc-name-type-spacing  --no-numpydoc-name-type-spacing]
                                 [files ...]
 
     positional arguments:
       files                 The directory or files to format.
 
-    options:
+    optional arguments:
```

Comment:
Which interpreter version are you using locally?

Reply:
Cygwin CPython 3.9; should I be using 3.8 or 3.7 for this?

Reply:
Was hoping it might be because I did `pip install -r requirements-test.txt` instead of `pip install -U -r requirements-test.txt`, but adding the -U didn't seem to change anything